cg/services/delivery_message/messages/fastq_message.py

```diff
@@ -4,8 +4,8 @@
 
 
 class FastqMessage(DeliveryMessage):
-    def create_message(self, case: Case) -> str:
-        delivery_path: str = get_fastq_delivery_path(case)
+    def create_message(self, cases: list[Case]) -> str:
+        delivery_path: str = get_fastq_delivery_path(cases[0])
```

Comment:
will this not throw an index error when the list is empty?

Reply:
Should it not throw an error in that case? @ChrOertlin

Reply:
I think the error that would be thrown would be in "index out of range" error. Maybe we can try/except with a clearer message?

Reply:
I think there will be no use case where that happens. And using this endpoint with no case_id at all will yield an error saying that the internal_ids were not found in the database. Probably good enough?